build/LayoutNugetPackage.proj

```diff
@@ -47,6 +52,10 @@
     <SourceFile Include="..\src\ADAL.PCL.Android\bin\$(Configuration)\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.pdb">
       <TargetFolder>MonoAndroid10</TargetFolder>
     </SourceFile>
+    <SourceFile Include="..\src\ADAL.PCL.Android\bin\$(Configuration)\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.xml"
+                Condition="$(CopyXmlDocFiles)=='true'">
+      <TargetFolder>MonoAndroid10</TargetFolder>
+    </SourceFile>
```

Comment:
Note that the .Net Core project doesn't produce an XML file so there isn't a _SourceFile_ entry for it.